modules/cnv-creating-vddk-image.adoc

```diff
@@ -0,0 +1,67 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv_users_guide/cnv-importing-vmware-vm.adoc
+
+[id="cnv-creating-vddk-image_{context}"]
+= Creating the Virtual Disk Development Kit image
+
+You can download the VMware Virtual Disk Development Kit (VDDK) SDK, convert it into an image, and push it to a container repository.
+
+.Prerequisites
+
+* *My VMware* credentials to download the VDDK SDK
+* Private container repository. Storing the VDDK SDK in a public repository may violate the terms of the VMware license.
+
+.Procedure
+
+. Create a temporary directory for VDDK:
++
+----
+$ mkdir /tmp/my_vddk && cd /tmp/my_vddk
+----
+
+. Download the VDDK SDK:
+.. Navigate to link:https://www.vmware.com/support/pubs/[VMware Documentation].
+.. Click *VMware SDK & API Product Documentation* -> *VMware Virtual Disk Development Kit (VDDK)*.
+.. Select the latest VDDK release and click *Download SDKs* to download the `tar.gz` archive.
```

Comment:
s/Select the latest VDDK release and/Select the latest VDDK release, and then/

Reply:
I'm willing to add "then" in case some poor user tries to select a release and click a button at the same time but a serial comma is unnecessary in a series of two items.